config/test.json

```diff
@@ -49,5 +49,17 @@
   "logging": {
     "level": "error",
     "fmt": "pretty"
-  }
+  },
+  "serviceClients": [
+    {
+      "id": "d23dbf62b82eb04e",
+      "name": "Test Service Client",
+      "scope": "profile:email",
+      "pubkey": {
+        "kty": "RSA",
+        "n":"xaQHsKpu1KSK-YEMoLzZS7Xxciy3fsGrhrrqW_JBrq3IRmeGLaqlE80zcpIVnStyp9tbet2niYTemt8ug591YWO5Y-S0EgQyFTxnGjzNOvAL6Cd2iGie9QeSehfFLNyRPdQiadYw07fw-h5gweMpVJv8nTgS-Bcorlw9JQM6Il1cUpbP0Lt-F_5qrzlaOiTEAAb4JGOusVh0n-MZfKt7w0mikauMH5KfhflwQDn4YTzRkWJzlldXr1Cs0ZkYzOwS4Hcoku7vd6lqCUO0GgZvkuvCFqdVKzpa4CGboNdfIjcGVF4f1CTQaQ0ao51cwLzq1pgi5aWYhVH7lJcm6O_BQw",
+        "e":"AQAB"
```

Comment:
We'll need a way to support rotating these keys, so at the very least, it should probably be a set of keys rather than a single item.  In the absence of reasons to deviate from spec, we could use a JWK set [1].

I know I harp on about it, but I think we should consider using JKUs [2] here, and fetching the public keys dynamically rather than embedding them directly in the config.  This seemed to work nicely for preVerifyTokens in the auth-server, and provided a nice decoupling between systems.

Pros:
* decouples services from our config here, we just store the JKU value that we expect to see from them, and look up the actual key material dynamically
* services can rotate their keys without us having to change our config at all

Cons:
* Will probably make testing and local development harder, as you have to be able to fetch the keys
* Reduces the security of the system to that of TLS (although we can mitigate with cert pinning, and by ensuring the services don't communicate over the open internet)

@seanmonstar I'd be interested in your thoughts either way.

[1] http://tools.ietf.org/html/rfc7517#section-5
[2] http://tools.ietf.org/html/rfc7515#section-4.1.2